dockerfile

```diff
@@ -76,6 +77,11 @@ WORKDIR $HOME
 
 # Set env to tmpfs path
 ENV WALLET /home/node/tmp/wallet.json
+# Set testing ENV variables to sample IDs
+ENV PUB_DRIVE_ID "00000000-0000-0000-0000-000000000000"
+ENV PUB_FOLD_ID "00000000-0000-0000-0000-000000000000"
```

Comment:
Would it be problematic to use 1 and 2 for these to disambiguate the three of them?

Reply:
Nope, lets do it